Packs/CortexResponseAndRemediation/Playbooks/silent-playbook-Suspicious_Local_User_Account_Creation.yml

```diff
@@ -0,0 +1,2395 @@
+      value:
+        simple: Resolved - True Positive
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 448,
+          "y": 2024
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "41":
+    id: "41"
+    taskid: 40fa80a5-652e-45fe-90f4-e225de10248a
+    type: title
+    task:
+      id: 40fa80a5-652e-45fe-90f4-e225de10248a
+      version: -1
+      name: Remediation
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "45"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 448,
+          "y": 2154
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "43":
+    id: "43"
+    taskid: 53babb1d-1c09-4857-8f6d-1c71deecbad2
+    type: regular
+    task:
+      id: 53babb1d-1c09-4857-8f6d-1c71deecbad2
+      version: -1
+      name: Close Investigation
+      description: Close the current alert.
+      script: Builtin|||closeInvestigation
+      type: regular
+      iscommand: true
+      brand: Builtin
+    scriptarguments:
+      closeReason:
+        complex:
+          root: CloseReason
+          transformers:
+          - operator: SetIfEmpty
+            args:
+              applyIfEmpty: {}
+              defaultValue:
+                value:
+                  simple: Resolved - False Positive
+      id:
+        simple: ${alert.id}
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 2972
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "44":
+    id: "44"
+    taskid: ce712cfe-c409-46bf-9d4a-9e1a54c57b96
+    type: condition
+    task:
+      id: ce712cfe-c409-46bf-9d4a-9e1a54c57b96
+      version: -1
+      name: Analyst review - disable suspicious user?
+      description: |
+        We found suspicious indicators in this investigation and have taken action by disabling the suspicious local user that was created.
+
+        Review the findings and decide on the next steps:
+
+        ### Alert Information
+        The subject (creator) user: `${Core.OriginalAlert.event.normalized_evtlog_subject_user}`
+
+        The target (created) user: `${Core.OriginalAlert.event.evtlog_target_username}`
+
+        Involved Host: `${alert.hostname}`
+
+
+        ---
+
+        ### Related Alerts
+        Suspicious related alerts found: `${.=val.foundIncidents && val.foundIncidents.length > 0 ? "True" : "False"}`
```

Comment:
We should provide the related alerts found. There is not much value for the analyst when we say "Related alerts: True". He should know that the same user has potentially performed "Suspicious disablement of the Windows Firewall" for example.